LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/EC/EmbeddedController.cs

```diff
@@ -49,9 +49,9 @@ private enum ECSensor
             { ECSensor.TempMB, new EmbeddedControllerSource("Motherboard", SensorType.Temperature, 0x003c) },
             { ECSensor.TempTSensor, new EmbeddedControllerSource("T Sensor", SensorType.Temperature, 0x003d, blank: -40) },
             { ECSensor.TempVrm, new EmbeddedControllerSource("VRM", SensorType.Temperature, 0x003e) },
-            { ECSensor.FanCPUOpt, new EmbeddedControllerSource("CPU Optional", SensorType.Fan, 0x00b0, 2) },
-            { ECSensor.FanVrmHS, new EmbeddedControllerSource("VRM Heat Sink", SensorType.Fan, 0x00b2, 2) },
-            { ECSensor.FanChipset, new EmbeddedControllerSource("Chipset", SensorType.Fan, 0x00b4, 2) },
+            { ECSensor.FanCPUOpt, new EmbeddedControllerSource("CPU Optional Fan", SensorType.Fan, 0x00b0, 2) },
```

Comment:
They are in the "Fans" section, why repeating that in names?

Reply:
I see.

Reply:
I think it's done because when you also have a "control" sensor for it, you know what it's referring to, it would be a bit odd to have a control for "External". Although perhaps Control elements can retain Fan and remove it from Fan sensors. Hmmm 🤔

Reply:
Let me also note that not only fans can be connected to these headers, but pumps too.